grunt_tasks/build.js

```diff
@@ -77,17 +77,11 @@ module.exports = function (grunt) {
                     dest: 'clients/web/static/built/jsoneditor'
                 }]
             },
-            bootstrap: {
+            // Provide a copy of ext.min.js under an old name, for compatibility
+            legacy_libs: {
                 files: [{
-                    expand: true,
-                    cwd: 'node_modules/bootstrap/dist/css',
-                    src: ['bootstrap.min.css'],
-                    dest: 'clients/web/static/built'
-                }, {
-                    expand: true,
-                    cwd: 'node_modules/bootstrap-switch/dist/css/bootstrap3',
-                    src: ['bootstrap-switch.min.css'],
-                    dest: 'clients/web/static/built'
+                    src: ['clients/web/static/built/ext.min.js'],
+                    dest: 'clients/web/static/built/libs.min.js'
```

Comment:
Windows has supported symlinks (and hardlinks, too) since XP.  The command
mklink has existed since Vista.

On Mon, Jan 11, 2016 at 3:44 PM, Zach Mullen <notifications@github.com>
wrote:

> In grunt_tasks/build.js
> <https://github.com/girder/girder/pull/1161#discussion_r49375928>:
>
> >                  files: [{
> > -                    expand: true,
> > -                    cwd: 'node_modules/bootstrap/dist/css',
> > -                    src: ['bootstrap.min.css'],
> > -                    dest: 'clients/web/static/built'
> > -                }, {
> > -                    expand: true,
> > -                    cwd: 'node_modules/bootstrap-switch/dist/css/bootstrap3',
> > -                    src: ['bootstrap-switch.min.css'],
> > -                    dest: 'clients/web/static/built'
> > +                    src: ['clients/web/static/built/ext.min.js'],
> > +                    dest: 'clients/web/static/built/libs.min.js'
>
> We don't support windows for the server. I believe modern windows has
> symlinks with at least some level of support, plus windows users could just
> update to the new path.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/girder/girder/pull/1161/files#r49375928>.
>



-- 
David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439